numpy/core/src/multiarray/arraytypes.c.src

```diff
@@ -1530,7 +1530,7 @@ OBJECT_to_@TOTYPE@(void *input, void *output, npy_intp n,
  * #convert = 1*18, 0*3, 1*2,
  *            1*18, 0*3, 1*2,
  *            0*23#
- * #convstr = (Int*9, Long*2, Float*4, Complex*3, Tuple*3, Long*2)*3#
+ * #convstr = (Long*9, Long*2, Float*4, Complex*3, Tuple*3, Long*2)*3#
```

Comment:
#define PyInt_type PyLong_type changes these Int's to Long's

took me a second to understand this logic (also to see if I could remove IS_STRING)